manifests/00-cluster-role.yaml

```diff
@@ -59,6 +60,18 @@ rules:
   - watch
   - update
 
+# Add escalate and bind verbs for role/rolebindings.
+# This allows the operator to grant broad route permissions to itself in the ingress-canary namespace
+# without holding the same permissions in the operator namespace.
```

Comment:
Does this mean that the operator now has the ability to create an arbitrary clusterrole and then bind its own serviceaccount to that clusterrole?  If so, this effectively gives the operator unbound permissions.